### ISSUE TITLE
fictional child

### ISSUE THREAD

**Issue by eatyourpeas** [rcpch/digital-growth-charts-server#223]
It is occasionally helpful for other apps to have growth data of fictional children that make sense. The fiction-child-data endpoint produces values as a list, but occasionally the user only needs one or two.

Perhaps instead we could have two endpoints - one which returns a single value, and another which returns a list.

**Comment by statist7** [comment]
Why not just specify the number you want, defaulting to one?

**Comment by eatyourpeas** [comment]
yes exactly. Currently the endpoint accepts a start age and an end age, with optional parameters for drift, randomisation and so on. So perhaps just one endpoint then but make it a little easier for the user to request just one value

**Comment by pacharanero** [comment]
Closing for now: the existing fictional-child-data endpoints already generate measurement series, and we have not identified a demonstrated user or customer need for an explicit measurement-count parameter or a separate single-measurement endpoint. This is a potential enhancement rather than a confirmed requirement. We can reopen this issue if a concrete use case demonstrates that the existing age-range and interval controls are insufficient.
